Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```

Comment:
We probably want to remove this to avoid accidental committing.

Reply:
Fixed